.github/workflows/actions.yml

```diff
@@ -6,6 +6,20 @@ on:
     - master
 
 jobs:
+  dependency-guard:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Block dependencies from being added to package.json
+        run: |
+          IFS=' ' read -r allowed <<< "native-promise-only zendesk_app_framework_sdk"
```

Comment:
That's largely already done by GHA I believe: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
In this case, we should define the shell to be bash I believe, since the undefined only uses `bash -e {0}`

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
👍